capstone/src/components/User/Adventurer/FindAdventure/Map/Map.jsx

```diff
@@ -61,10 +68,33 @@ export default function Map() {
           >
             {activeRestaurant === restaurant.place_id ? (
               <InfoWindowF position={restaurant.geometry.location}>
-                <Box width="120px">
+                <Box
+                  width="120px"
+                  display="flex"
+                  justifyContent="center"
+                  flexDirection="column"
+                >
                   <Heading size="l">{restaurant.name}</Heading>
                   <Text size="l">{restaurant.formatted_address}</Text>
-                  <Badge colorScheme="purple">{restaurant.matchScore}% match</Badge>
+                  <Image
+                    width="auto"
+                    height="70px"
+                    src={`https://maps.googleapis.com/maps/api/place/photo?maxwidth=400&photo_reference=${
+                      restaurant.photos
+                        ? restaurant.photos[0].photo_reference
+                        : null
+                    }&key=${API_KEY}`}
+                  />
+                  {restaurant.matchScore ? (
+                    <Badge mt="5px" colorScheme="purple">
+                      {restaurant.matchScore}% match
+                    </Badge>
+                  ) : null}
+                  {restaurant.reviewsNumber ? (
+                    <Badge mt="5px" colorScheme="blue">
+                      {restaurant.reviewsNumber} reviews
+                    </Badge>
+                  ) : null}
```

Comment:
These JSX components are looking a lot easier to read, nice!

Reply:
Thanks for that!